dhctl/pkg/kubernetes/actions/deckhouse/delete.go

```diff
@@ -49,6 +61,31 @@ func DeleteDeckhouseDeployment(kubeCl *client.KubernetesClient) error {
 	})
 }
 
+func DeleteDeckhouseStorageCRs(kubeCl *client.KubernetesClient) error {
+	return retry.NewLoop("Delete Deckhouse Storage CRs", 45, 5*time.Second).WithShowError(false).Run(func() error {
+		for _, cr := range d8storageConfig.Resources {
+			resourceSchema := schema.GroupVersionResource{Group: d8storageConfig.ApiGroup, Version: d8storageConfig.ApiVersion, Resource: cr}
+			storageCRs, err := kubeCl.Dynamic().Resource(resourceSchema).Namespace(metav1.NamespaceAll).List(context.TODO(), metav1.ListOptions{})
```

Comment:
Instead of context.TODO(), context.WithTimeout() can and probably should be used